types/gtag.js/index.d.ts

```diff
@@ -91,4 +98,14 @@ declare namespace Gtag {
     id?: string;
     name?: string;
   }
+
+  type FieldNames = 'client_id' | 'session_id' | 'gclid';
+
+  type ConsentArg = 'default' | 'update';
+  interface ConsentParams {
+    ad_storage?: 'allowed' | 'denied';
```

Comment:
Fixed : https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52148#discussion_r612526045

Reply:
Should be updated - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52148#discussion_r635145966